.gitignore

```diff
@@ -4,3 +4,5 @@ dev_dmdi_files/
 .DS_Store
 __pycache__/
 tmp/
+generate_files.pkexec
```

Comment:
Good job! I have 3 questions: 
- have you tested it? I assume you did, but just in case
- We don't really need units such as inches in prettyprinter, but I think you have kept them for completeness while you were translating the TARALLO code into Python, right?
- Why have you excluded this file? As I remember it, it's needed in the GUI to prompt the user for the password in the DE.

Reply:
* Yes, but I tested the GUI part with the old GUI which doesn't work properly anymore, so I couldn't really test it. However there are automated tests on the part I added there, so we should be good
* Yes, exactly. The screen diagonal in inches could be in EDID data, which we don't parse currently, but it could make sense to parse in the future.
* Yes asd